examples/AutoFilterTutorial.0.0.cpp

```diff
@@ -0,0 +1,142 @@
+// AutoFilterTutorial.0.0.cpp created by Victor Dods on 2015/06/03
```

Comment:
Git history already documents the creator of a file, there's no need to place a redundant contributor notice here.